stdlib/public/Platform/Windows/SwiftWinDbg/SwiftWinDbg.cpp

```diff
@@ -0,0 +1,109 @@
+#define WIN32_LEAN_AND_MEAN // Exclude rarely-used stuff from Windows headers
+#include <windows.h>
+#define KDEXT_64BIT
+#include <dbgeng.h>
+#include <wdbgexts.h>
+#include <stdint.h>
+
+WINDBG_EXTENSION_APIS ExtensionApis = {0};
+EXT_API_VERSION g_ExtApiVersion = {1, 0, EXT_API_VERSION_NUMBER64, 0};
+USHORT SavedMajorVersion, SavedMinorVersion;
+
+bool APIENTRY DllMain(HMODULE hModule, DWORD ul_reason_for_call,
+                      LPVOID lpReserved) {
+  switch (ul_reason_for_call) {
+  case DLL_PROCESS_ATTACH:
+  case DLL_THREAD_ATTACH:
+  case DLL_THREAD_DETACH:
+  case DLL_PROCESS_DETACH:
+    break;
+  }
+  return true;
+}
```

Comment:
Seems like we should be able to completely drop this?  The default `DllMain` should be equivalent.